frontend/src/metabase/dashboard/components/DashboardHeader.jsx

```diff
@@ -256,27 +238,11 @@ export default class DashboardHeader extends Component {
             );
 
             buttons.push(
-                <ModalWithTrigger
-                    key="history"
-                    ref="dashboardHistory"
-                    triggerElement={
-                        <Tooltip tooltip="Revision history">
-                            <span data-metabase-event={"Dashboard;Revisions"}>
-                                <Icon className="text-brand-hover" name="history" size={18} />
-                            </span>
-                        </Tooltip>
-                    }
-                >
-                    <HistoryModal
-                        entityType="dashboard"
-                        entityId={dashboard.id}
-                        revisions={this.props.revisions["dashboard-"+dashboard.id]}
-                        onFetchRevisions={this.onFetchRevisions.bind(this)}
-                        onRevertToRevision={this.onRevertToRevision.bind(this)}
-                        onClose={() => this.refs.dashboardHistory.toggle()}
-                        onReverted={() => this.onRevertedRevision()}
-                    />
-                </ModalWithTrigger>
+                <Tooltip tooltip="Revision history">
+                    <Link to={location.pathname + "/history"} data-metabase-event={"Dashboard;Revisions"}>
```

Comment:
It'd be nice if we could come up with a slightly cleaner way to allow for relative URLs like this instead of having to use `location.pathname`

Reply:
I think that in this case a little nicer way would be adding `dashboardRevisionHistory(dashboardId)` to `urls.js` and calling that instead of using a relative url.

React-router doesn't seem to provide a clean syntax for relative urls (they discuss it in https://github.com/ReactTraining/react-router/issues/5127 for v4 but there's no implementation yet).